### PR TITLE
Type: fill --> file

### PR DESCRIPTION
**Pull request by elharo**

### CODE REVIEW

File: src/site/apt/index.apt.vm [hunk 1]
```diff
@@ -52,7 +52,7 @@ ${project.name}
   already contain the answer to your question as part of an older thread. Hence, it is also worth browsing/searching
   the {{{./mailing-lists.html}mail archive}}.
 
-  If you feel like the plugin is missing a feature or has a defect, you can fill a feature request or bug report in our
+  If you feel like the plugin is missing a feature or has a defect, you can file a feature request or bug report in our
```
Comment (gnodet) [inline src/site/apt/index.apt.vm:55]: submit ?

Comment (elharo) [reply src/site/apt/index.apt.vm:55]: as soon as someone approves

Comment (gnodet) [reply src/site/apt/index.apt.vm:55]: > submit ?

I meant as another suggestion for the `fill` / `file` wording.

Comment (elharo) [reply src/site/apt/index.apt.vm:55]: Oh, I get it now. Yes, that is better wording.

**Review by gnodet — APPROVED** [review]

**Pull request merged**